### DISCUSSION TITLE
Power, Voltage, Current values shop showing when disconnected from internet.

### DISCUSSION THREAD

**Discussion by AlexK98** [make-all/tuya-local#2711] [Q&A]
Hello,

I have a few smart circuit breakers ((Tomzn TOMPD-63LW, Tomzn TOMPD-63WIFI) and energy meters (TAIYEDQ TAC2121C BI, XOCA DAC2121C BI). Today, my main internet connection line was down and during the downtime, DPs like Power, Voltage and Current (and other frequently updated data) from these devices were showing as "Unknown".

![image](https://github.com/user-attachments/assets/1b4cce64-f3fe-4622-8818-1bf554b3f379)

IPs for all the devices are static.

Why would/could devices stop sending these DPs?
They are not 'cloud' DPs.

**Comment by Ivkaiv** [comment]
I have the same problem with Taxnele

**Comment by Deapper** [comment]
Also taxnele same problem

**Comment by make-all** [comment]
Only the device manufacturer can explain device behaviour issues like this.
